pyscitt/pyscitt/client.py

```diff
@@ -369,14 +369,13 @@ def submit_claim(
 
     @overload
     def submit_claim(
-        self, claim: bytes, *, skip_confirmation: Literal[True]
+        self, claim: bytes, *, skip_confirmation: Literal[True], headers = {"Content-Type": "application/cose"}
     ) -> PendingSubmission:
         ...
 
     def submit_claim(
-        self, claim: bytes, *, skip_confirmation=False
+        self, claim: bytes, *, skip_confirmation=False, headers = {"Content-Type": "application/cose"}
```

Comment:
Rather than having this as a default value, I think we should take whatever the caller passed and add the "content-type" header. Otherwise, a call with `submit_claim(..., headers={  "x-ms-client-request-id": "foo" })` would be missing the content type.

Reply:
Good point. Change to update instead of replace header.